tests/scalar_advection_diffusion/scalar_advection_diffusion.py

```diff
@@ -58,27 +58,28 @@
 dt = T/600.0
 
 
-# Use G-ADOPT's Energy Solver to advect the tracer. By setting the Rayleigh number to 1
-# the choice of units is up to the user. We use the diagonaly implicit DIRK33 Runge-Kutta
-# method for timestepping. 'T' means that the boundary conditions will be applied strongly
-# by the energy solver.
-approximation = BoussinesqApproximation(Ra=1, kappa=kappa)
+# Use G-ADOPT's GenericTransportSolver to advect the tracer. We use the diagonally
```

Comment:
See previous comment.

Reply:
Sorry you are right. I was getting confused with different test cases. Since we still have users (earthbyte) using SU with our energy solver, I just want to make sure that this aspect is also tested.

Reply:
This time, it is an advection-diffusion problem, but again, it does not have much to do with the usual energy conservation we usually refer to. Moreover, there is no reason to involve an approximation at all. A simple, generic solver can perfectly handle the situation.